spec/README.md

```diff
@@ -95,12 +95,141 @@ __Deliverables__:
 
 
 ## D. Interface for Traffic Information
-TODO
+__Goal__: To build a way for QGCS to receive real-time traffic information, and display concerning traffic in the QGCS user interface.
+
+__Prerequisites:__
+- Capability to determine authenticity of traffic information received
```

Comment:
Thanh you for this, I wonder if this can be v2 feature but perhaps it is in built in QUIC

Reply:
For information security, I believe it would suffice to assert that the UTM service providers are who they say they are. Authentication is part of establishing a QUIC connection (QUIC comes with TLS included).

The UTM service provider receives network information protected by TLS, and ADS-B/1090 MHz traffic information protected by law.

To determine if the original information sources were authentic would be out of scope of a ground control station. That is not common practice AFAIK. 

IMO, using QUIC should tick that box.